index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Survey Form</title>
+    <link rel="stylesheet" href="css/style.css">
+    
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet">
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+  </head>
+
+  <body>
+    <main>
+      <h1>
+        freeCodeCamp Survey Form
+      </h1>
+
+      <p>
+        Thank you for taking the time to help us improve the platform
+      </p>
+
+      <form action="" id="survey-form" class="survey-form">
+        <label for="">Name</label>
+        <input type="text" name="name" id="name" placeholder="Enter your name">
+
+        <label for="">Email</label>
+        <input type="email" name="email" id="email" placeholder="Enter your Email">
+
+        <label for="">Age <small>(optional)</small></label>
+        <input type="number" name="name" id="age" placeholder="Enter your Age">
+
+        <label for="">Which option best describes your current role?</label>
+        <select name="" id="">
+          <option value="">Student</option>
+          <option value="">Full Time Job</option>
+          <option value="">Full Time Learner</option>
+          <option value="">Prefer not to say</option>
+          <option value="">Other</option>
+        </select>
+        
+        <label for="">Would you recommend freeCodeCamp to a friend?</label>
+        <input type="radio" name="" id="">Definately
+        <input type="radio" name="" id="">Maybe
+        <input type="radio" name="" id="">Not sure
+
+        <label for="">What is your favorite feature of freeCodeCamp?</label>
+        <!-- <select name="" id="">
+          <option value="">Challenges</option>
+          <option value="">Communities</option>
+          <option value="">Projects</option>
+          <option value="">Open Source</option>
+        </select> -->
```

Comment:
Please don't put unnecessary code as a commet, so please remove it because it's make your code looking not clean🙂